packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTable/CWTable.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../../../../styles/shared.scss';
+
+.Table {
+  table {
```

Comment:
Can we make it fullwidth? On empty data it shows like this

<img width="1212" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/b58a3fcc-3b96-48a6-baf6-721ab99c129b">

Reply:
Good catch! Table is now fullwidth regardless of presence of data.